llpc/include/llpc.h

```diff
@@ -232,6 +232,28 @@ class ICompiler {
   /// @returns : Result::Success if successful. Other return codes indicate failure.
   virtual Result BuildShaderModule(const ShaderModuleBuildInfo *pShaderInfo, ShaderModuleBuildOut *pShaderOut) = 0;
 
+  /// Build shader from the specified info.
+  ///
+  /// @param [in]  pipelineInfo    Info to build this shader module
+  /// @param [out] pipelineOut : Output of building this shader module
+  /// @param [in]  stage          Shader stage of needing to compile
+  /// @param [out] pipelineDumpFile : Handle of pipeline dump file
+  ///
+  /// @returns : Result::Success if successful. Other return codes indicate failure.
+  virtual Result buildGraphicsShaderStage(const GraphicsPipelineBuildInfo *pipelineInfo,
+                                          GraphicsPipelineBuildOut *pipelineOut, Vkgc::UnlinkedShaderStage stage,
+                                          void *pPipelineDumpFile = nullptr) = 0;
```

Comment:
nit: `pipelineDumpFile`? I think the `p` prefix is not necessary anymore

Reply:
It seems 'p' prefix is added for the pointer type in this file, I keep the same format. Thanks.